### ISSUE TITLE
Gradle 7+ compatibility

### ISSUE THREAD

**Issue by JoshStark** [michaelruocco/environment-properties-plugin#1]
Hi, 

I'm not sure how maintained this project is but it's something I'd very much like to use. Unfortunately while using the latest Gradle 7.+ release, it errors with the following:

```
A problem was found with the configuration of task ':generateMultipleEnvironmentProperties' (type 'GenerateMultipleEnvironmentPropertiesTask').
  - In plugin 'com.github.michaelruocco.environment-properties' type 'uk.co.mruoc.plugin.environment.properties.GenerateMultipleEnvironmentPropertiesTask' method 'buildPropertiesFile()' should not be annotated with: @Internal.

    Reason: Input/Output annotations are ignored if they are placed on something else than a getter.

    Possible solutions:
      1. Remove the annotations.
      2. Rename the method.

    Please refer to https://docs.gradle.org/7.2/userguide/validation_problems.html#ignored_annotations_on_method for more details about this problem.
```

Would it be possible for the task annotations to be updated to account for this?
Cheers.

**Comment by michaelruocco** [comment]
Hi Josh, how are you?

Yeah that should be doable I think, I've just had a quick look and noticed I've not update this repo for a good while so its still using Travis CI and not automatically publishing to nexus / gradle plugin repository. I'll have a look at updating those at some point.

I probably wont get time until the weekend to look at it properly but I'll try and implement the changes required then. If you need it sooner than that then feel free to send over a PR and Ill have a look at manually releasing it for you.

Thanks.

**Comment by michaelruocco** [comment]
I've just managed to have a stab at making the changes and its published version 0.1.6 and (0.1.7 after I updated the readme and removed the old travis CI pipeline and 0.1.8 after I updated a bunch of other dependencies): https://plugins.gradle.org/plugin/com.github.michaelruocco.environment-properties. Please give it a try and let me know if you still see any issues?

**Comment by JoshStark** [comment]
Hi Mike, I'm doing well, thanks! 

I've just updated to 0.1.8 and can now generate the properties files without warning or error, so thanks for the really quick turnaround with that!

I hope you're doing well. It's funny how I ended up back at this plugin when looking for something to handle environment properties.

**Comment by michaelruocco** [comment]
Ah great stuff, glad its worked. No worries, I've not had much chance to work on any of my own repos recently so it was good to have an excuse to have a look at one of them again.

Yeah I am good thanks. Its a shameless rip off of the functionality we used to have in the web services plugin at TP. I had a work project a few years back where we had a similar issue with a number of the properties being duplicated across environments so we made use of it for a while.
